### PR TITLE
Add designated property for content security policy

### PR DESCRIPTION
**Pull request by ko5tik**
This pull request adds property for content security policy configuration as discussed with Frederik ( @atomfrede )   in:

https://github.com/jhipster/generator-jhipster/pull/14396

### CODE REVIEW

File: jhipster-framework/src/main/java/tech/jhipster/config/JHipsterDefaults.java [hunk 1]
```diff
@@ -126,7 +126,7 @@
     }
 
     interface Security {
-
+        String contentSecurityPolicy = "content-security-policy: \"default-src 'self'; frame-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://storage.googleapis.com; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self' data:";
```
Comment (deepu105) [inline jhipster-framework/src/main/java/tech/jhipster/config/JHipsterDefaults.java:129]: I'm not sure if this the correct format, I think should be only value

**Comment by ko5tik** [comment]
You are definitely correct.
========[] Lines game back in Appstore ß check it out! []===============
https://play.google.com/store/apps/details?id=de.pribluda.games.android.lines

Blog:  https://www,pribluda.de

File: jhipster-framework/src/main/java/tech/jhipster/config/JHipsterDefaults.java (hunk 1 above)
Comment (ko5tik) [reply jhipster-framework/src/main/java/tech/jhipster/config/JHipsterDefaults.java]: pushed corection

**Review by DanielFran — APPROVED** [review]

File: jhipster-framework/src/main/java/tech/jhipster/config/JHipsterDefaults.java [hunk 2]
```diff
@@ -126,7 +126,7 @@
     }
 
     interface Security {
-
+        String contentSecurityPolicy = "default-src 'self'; frame-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://storage.googleapis.com; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self' data:";
         interface ClientAuthorization {
```
Comment (DanielFran) [inline jhipster-framework/src/main/java/tech/jhipster/config/JHipsterDefaults.java:130]: Add an empty line

Comment (DanielFran) [reply jhipster-framework/src/main/java/tech/jhipster/config/JHipsterDefaults.java:130]:
```suggestion
        
        interface ClientAuthorization {
```

**Comment by DanielFran** [comment]
@ko5tik Thanks, will merge it as soon as the tests pass

**Comment by ko5tik** [comment]
@DanielFran commit must go before https://github.com/jhipster/generator-jhipster/pull/14396  can proceed

**Pull request merged**